scripts/utils.js

```diff
@@ -21,7 +21,7 @@ async function approveContractCall(
         ['string', 'string', 'address', 'bytes32', 'bytes32', 'uint256'],
         [sourceChain, sourceAddress, contractAddress, keccak256(payload), sourceTxHash, sourceEventIndex],
     );
-    await (await gateway.approveContractCall(params, commandId)).wait();
+    await gateway.approveContractCall(params, commandId).then((tx) => tx.wait);
```

Comment:
```suggestion
    await gateway.approveContractCall(params, commandId).then((tx) => tx.wait());
```
missing function call